.gitignore

```diff
@@ -12,3 +12,5 @@ testing/
 /out/build
 /CMakeSettings.json
 /out/mytest
+gdb.txt
+.gdb_history
```

Comment:
I'm not quite sure whether we want to add more stuff here.

Reply:
gdb.txt is the default filename of the log file generated by gdb option 'set logging on'
.gdb_history is the default filename of the history file generated by the gdb option 'set history save on'.

Other than a minor annoyance, it doesn't matter much to me whether or not these are added to .gdbignore.  Removed.

Reply:
You can use `.git/info/exclude` locally